specification/dimensions/pricingunit.md

```diff
@@ -0,0 +1,31 @@
+# Pricing Unit
+
+The Pricing Unit represents a provider specified pricing measurement unit, which forms the foundation for determining unit prices of utilizing a particular resource and/or service. Common examples include the number of hours a compute appliance was running for a given period, the number of gigabytes * the number of hours (gigabyte-hours) for a storage appliance, or a cumulative count of requests for a network appliance or API service. For certain services providers can choose block pricing i.e., million requests as a base unit.
+PricingUnit is complementary to and describes how [QuantityInPricingUnit](#quantityinpricingunit) is measured.
+
+The PricingUnit column MUST be present in the billing data. This column MUST be of type String. It MUST NOT contain null if [QuantityInPricingUnit](#quantityinpricingunit) is not null. The PricingUnit value MUST match what is shown on the invoice if a pricing unit is present on the invoice.
```

Comment:
Is there not a standard set of units (or a combination) we're recommending like the usage unit team is doing? 

cc @amitw-g @ijurica

Reply:
I believe UsageQuantity + UsageUnit pair can also be described as a semi-normalized version of QuantityinPricingUnit + PricingUnit (not-normalized) pair.  I agree, we should mention that in the supporting content. As for the definition, phrasing this is a challenge :(.  I suggest we get back to that at a later stage, once names and definitions of UsageUnit and Quantity are finalized.

Few more notes about the PricingUnit...

**For AWS, Microsoft, GCP:**

PricingUnit comes down to mapped/renamed Provider's column

**For OCI:**

There is a column in billing data which we could use but it might be better to fetch the corresponding value from Public Pricing API (we already plan to use it for ListUnitPrice). 
Example:

- values in billing data: `ONE GiB HOURS STORAGE_SIZE`, `ONE GiB HOURS MEMORY`
- corresponding value in Public Pricing API: `Gigabyte Per Hour`

Reply:
We have discussed that it may be very hard to ship 1.0 with standard set of units, my believe is that we should keep examples as in the definition to get the idea of what is this column referring to. Later on we will see what is the value of those being standardised

Reply:
We have a set of recommended units, but the UsageUnit was transitioned from a "MUST use Allowed Values" to "SHOULD use recommended values" - maybe we should move that part into an appendix and refer to it from both?

Reply:
- notes regarding non-normalized vs normalized and future plans provided within supporting content
- inclusion of PricingUnit - UsageUnit cross-references in progress
- current values observed in billing data for OCI provided